src/widgets/block.rs

```diff
@@ -88,6 +91,11 @@ impl<'a> Block<'a> {
         self
     }
 
+    pub fn clean(mut self, clean: bool) -> Block<'a> {
+        self.clean = clean;
+        self
+    }
+
```

Comment:
```suggestion
    pub fn clean(mut sel) -> Block<'a> {
        self.clean = true;
        self
    }
```
Maybe this is better?